modules/library/main/src/main/java/org/geotools/util/ColorConverterFactory.java

```diff
@@ -43,66 +46,259 @@
+        colors.put("navajowhite", new Color(255, 222, 173));
+        colors.put("navy", new Color(0, 0, 128));
+        colors.put("oldlace", new Color(253, 245, 230));
+        colors.put("olive", new Color(128, 128, 0));
+        colors.put("olivedrab", new Color(107, 142, 35));
+        colors.put("orange", new Color(255, 165, 0));
+        colors.put("orangered", new Color(255, 69, 0));
+        colors.put("orchid", new Color(218, 112, 214));
+        colors.put("palegoldenrod", new Color(238, 232, 170));
+        colors.put("palegreen", new Color(152, 251, 152));
+        colors.put("paleturquoise", new Color(175, 238, 238));
+        colors.put("palevioletred", new Color(219, 112, 147));
+        colors.put("papayawhip", new Color(255, 239, 213));
+        colors.put("peachpuff", new Color(255, 218, 185));
+        colors.put("peru", new Color(205, 133, 63));
+        colors.put("pink", new Color(255, 192, 203));
+        colors.put("plum", new Color(221, 160, 221));
+        colors.put("powderblue", new Color(176, 224, 230));
+        colors.put("purple", new Color(128, 0, 128));
+        colors.put("rebeccapurple", new Color(102, 51, 153));
+        colors.put("red", new Color(255, 0, 0));
+        colors.put("rosybrown", new Color(188, 143, 143));
+        colors.put("royalblue", new Color(65, 105, 225));
+        colors.put("saddlebrown", new Color(139, 69, 19));
+        colors.put("salmon", new Color(250, 128, 114));
+        colors.put("sandybrown", new Color(244, 164, 96));
+        colors.put("seagreen", new Color(46, 139, 87));
+        colors.put("seashell", new Color(255, 245, 238));
+        colors.put("sienna", new Color(160, 82, 45));
+        colors.put("silver", new Color(192, 192, 192));
+        colors.put("skyblue", new Color(135, 206, 235));
+        colors.put("slateblue", new Color(106, 90, 205));
+        colors.put("slategray", new Color(112, 128, 144));
+        colors.put("slategrey", new Color(112, 128, 144));
+        colors.put("snow", new Color(255, 250, 250));
+        colors.put("springgreen", new Color(0, 255, 127));
+        colors.put("steelblue", new Color(70, 130, 180));
+        colors.put("tan", new Color(210, 180, 140));
+        colors.put("teal", new Color(0, 128, 128));
+        colors.put("thistle", new Color(216, 191, 216));
+        colors.put("tomato", new Color(255, 99, 71));
+        colors.put("turquoise", new Color(64, 224, 208));
+        colors.put("violet", new Color(238, 130, 238));
+        colors.put("wheat", new Color(245, 222, 179));
+        colors.put("white", new Color(255, 255, 255));
+        colors.put("whitesmoke", new Color(245, 245, 245));
+        colors.put("yellow", new Color(255, 255, 0));
+        colors.put("yellowgreen", new Color(154, 205, 50));
+        CSS_COLORS = Collections.unmodifiableMap(colors);
+    }
+
+    /**
+     * Converts CSS Color Module 4 names to colors.
+     */
+    private static Converter CONVERT_CSS_NAME_TO_COLOR = new Converter() {
```

Comment:
Yes, although this brings up the broader issue -- the CSS Color Module (and MapBox Styles, for that matter) support not only color names, but also strings in the format of rgba(x,x,x,x) / hsla(x,x,x,x). Don't want this name to be misleading.